src/server/entry.tsx

```diff
@@ -91,12 +92,21 @@ router.get('/:locale(se|se-en|no|no-en)/referrals/:code', async (ctx) => {
   ctx.status = 301
   ctx.redirect(`/${ctx.params.locale}/forever/${ctx.params.code}`)
 })
-redirects.forEach(([source, target, code]) => {
-  router.get(source, (ctx) => {
-    ctx.status = code
-    ctx.redirect(target)
+getRedirects()
+  .then(async (data) => {
+    appLogger.info(`redirect data ${JSON.stringify(data)}`)
+    data.map((datasource) => {
+      datasource.datasource_entries.map((redirect: DatasourceEntry, index) => {
+        router.get(redirect.name, (ctx) => {
```

Comment:
While this is clever it'll require us to restart the servers each time we change a redirect, ultimately leaving the ability to edit the data to developers anyways. I think rather we should write our own middleware that just gets the redirects and checks the path "run time" instead if "start time". Since we're using `cachedGet` anyways it should be fine to shoot that request every page load because it's stored in redis and redis is XtrEmelY speedy